lib/trento.pm

```diff
@@ -826,6 +828,46 @@ sub cluster_wait_status {
     die "Timeout waiting for the change" if !$done;
 }
 
+=head3 cluster_wait_status_by_regex
+
+Remotely run 'SAPHanaSR-showAttr' in a loop on $host, wait output that matches regular expression
+=over 3
+
+=item B<HOST> - Ansible name or filter for the remote host where to run 'SAPHanaSR-showAttr'
+
+=item B<TIMEOUT> - Max time to retry. Die if timeout
+
+=item B<REGULAR_EXPRESSION> - Regular expression to match the text to find
+
+=back
+=cut
+
+sub cluster_wait_status_by_regex {
+    my ($host, $regular_expression, $timeout) = @_;
+    $timeout //= bmwqemu::scale_timeout(300);
+    my $_monitor_start_time = time();
+    my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
+    my $done;
+    croak 'No regular expression provided' unless (ref $regular_expression eq 'Regexp');
+    while ((time() - $_monitor_start_time <= $timeout) && (!$done)) {
+        sleep 30;
+        my $show_attr = qesap_ansible_script_output(
+            cmd => 'SAPHanaSR-showAttr',
+            provider => $prov,
+            host => $host,
+            root => 1);
+
+        my @status = ();
```

Comment:
This line can be removed as the variable is no longer used.

Reply:
good catch thx